src/constants/cache.ts

```diff
@@ -0,0 +1,3 @@
+export const CACHE_NAME = "CPR_search_cache";
+export const CACHE_LIMIT = 100;
```

Comment:
Do you have an idea of how many bytes each cache entry is? ... if its ~10k then we could make the limit more like 1000 :shrug: ... take-it-or-leave-it :smile:

Reply:
According to this site: https://www.javainuse.com/bytesize
Each search entry is somewhere between 15 - 25kb, depending on the size of the result e.g. number of matches.

100 is a conservative limit. The maximum size of local storage is 5mb.

Reply:
Sounds good - let's not be one of those sites that take down your browser :laughing: